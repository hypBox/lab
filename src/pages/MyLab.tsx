import React from "react"
import Header from "../components/Header"

const myLab = () => {
  return (
    <>
      <Header />
      <h1>THis is MyLab</h1>
    </>
  )
}

export default myLab
