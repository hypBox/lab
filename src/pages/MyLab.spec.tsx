import React from "react"
import renderer from "react-test-renderer"

import MyLab from "./MyLab"

describe("Dashboard", () => {
  it("renders as expected", () => {
    const tree = renderer.create(<MyLab />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
