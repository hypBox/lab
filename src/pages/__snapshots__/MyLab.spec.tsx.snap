// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard renders as expected 1`] = `
<h1>
  THis is MyLab
</h1>
`;
